docs/organizations/audit/auditing-streaming.md

```diff
@@ -110,15 +110,16 @@ Once you have your Event Grid stream configured, you can set up subscriptions on
 ### Set up an Azure Monitor Log stream
 
 1. Create a [Log Analytics workspace](/azure/azure-monitor/learn/quick-create-workspace).
-2. Open the workspace and select **Agents management**.
-3. Make note of the workspace ID and primary key.
+2. Open the workspace and select **Agents**.
+3. Extend Log Analytics agent instructions
```

Comment:
I don't understand this step, please clarify, and then add a period at the end of the sentence. Thanks!

Reply:
Hi @chcomley, 114 step 3. When you open LAW the page is not visible with the keys and or Workspace ID. So I guess I can change that to: "Click on Log Analytics agent instructions drop down to open and view Workspace ID and Primary Key."